### PR TITLE
fix: do nothing for fixed widget option on variable deletion

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please review in order by commits

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_helpers/widget-validation-helper.ts [hunk 1]
```diff
@@ -172,15 +172,17 @@ const getAffectedWidgetInfoByDeletingVariableSchemaProperty = (
...
     dashboardVariables.forEach((variableKey) => {
         const property = getWidgetOptionName(variableKey);
         const isFixedProperty: boolean = widgetConfig.options_schema?.fixed_properties?.includes(property) ?? false;
 
-        if (isFixedProperty) { /* fixed property case */
-            set(_widgetOptions, property, get(widgetConfig.options, property, undefined)); // set default value to fixed option
-            _inheritOptions[property] = { enabled: false, variable_info: undefined }; // disable inherit option
-        } else { /* non-fixed property case */
+        // do nothing if inherit option is already disabled
+        if (!_inheritOptions[property]?.enabled) return;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_helpers/widget-validation-helper.ts:182]: praise: 🙌

**Review by yuda110 — APPROVED** [review]
LGTM!!!!!!

**Review by skdud4659 — APPROVED** [review]
LGTM~!

**Review by sulmoJ — APPROVED** [review]
LGT:M

**Pull request merged**